AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/CMakeLists.txt

```diff
@@ -145,6 +145,116 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
             AZ::AssetProcessor
     )
 
+    ly_add_pytest(
+        NAME AssetPipelineTests.Batch_2_Sandbox
+        PATH ${CMAKE_CURRENT_LIST_DIR}/asset_processor_batch_tests_2.py
+        EXCLUDE_TEST_RUN_TARGET_FROM_IDE
```

Comment:
I am curious why we are hiding this from the IDE. 
What is the purpose of EXCLUDE_TEST_RUN_TARGET_FROM_IDE?
It would seem we would like to encourage the community to discover and fix them, right?

Reply:
I don't know what that flag is for, it was used on other sandbox tests so I was just copy and pasting. I'm not very familiar with the pytest / cmake setup.

Reply:
I've verified it's not used for anything important and I've removed it